### PR TITLE
Update outside_controller_use.md

### PR DESCRIPTION
**Pull request by NickOttrando**
Provide example for options parameter when serializing a resource.

### CODE REVIEW

File: docs/howto/outside_controller_use.md [hunk 1]
```diff
@@ -10,8 +10,8 @@ In ActiveModelSerializers versions 0.10 or later, serializing resources outside
 # Create our resource
 post = Post.create(title: "Sample post", body: "I love Active Model Serializers!")
 
-# Optional options parameters
-options = {}
+# Optional options parameters for both the serializer and instance
+options = {serializer: PostDetailedSerializer, username: 'sample user'}
```
Comment (bf4) [inline docs/howto/outside_controller_use.md]: Let's mention they're the same options as passed to the controller and link to https://github.com/rails-api/active_model_serializers/blob/master/docs/general/rendering.md#explicit-serializer

Comment (NickOttrando) [inline docs/howto/outside_controller_use.md]: @bf4 added in f103d36

File: docs/howto/outside_controller_use.md [hunk 2]
```diff
@@ -20,6 +20,7 @@ serializable_resource = ActiveModelSerializers::SerializableResource.new(post, o
 model_json = serializable_resource.as_json
 ```
 The object that is passed to `ActiveModelSerializers::SerializableResource.new` can be a single resource or a collection.
+The additional options are the same options that are passed [through controllers](https://github.com/rails-api/active_model_serializers/blob/master/docs/general/rendering.md#explicit-serializer).
```
Comment (bf4) [inline docs/howto/outside_controller_use.md]: Let's use the relative link. `(../general/rendering.md#explicit-serializer)`

Comment (bf4) [inline docs/howto/outside_controller_use.md]: @NickOttrando Does this PR meet your needs of what you would have liked to have found before you arrived in the slack help channel?

Comment (NickOttrando) [inline docs/howto/outside_controller_use.md]: @bf4 edited to use relative link in last commit. Yeah I think this is clear and should hopefully help other people who may have had same confusion as me. Thanks

**Comment by bf4** [comment]
Perfect, awesome 💯

**Pull request merged**